src/core/Utils.js

```diff
@@ -524,6 +524,17 @@ function findDOMElement (element) {
   }
 }
 
+/**
+ * Find one or more DOM elements.
+ *
+ * @param {string} selector
+ * @return {Array|null}
+ */
+function findAllDOMElements (selector) {
+  const elements = [].slice.call(document.querySelectorAll(selector))
+  return elements.length > 0 ? elements : null
```

Comment:
🤔 looking at this now, `trigger:` should also still accept a DOM element being passed in. If it's not a CSS selector string but a DOM element we should return that in an array.